packages/interfaces/src/protocols.ts

```diff
@@ -22,18 +22,22 @@ export interface IBaseProtocol {
 
 export type ProtocolCreateOptions = {
   /**
-   * The PubSub Topic to use. Defaults to {@link @waku/core.DefaultPubSubTopic }.
+   * Waku supports usage of multiple pubsub topics, but this is still in early stages.
+   * Waku implements sharding to achieve scalability
+   * The format of the sharded topic is `/waku/2/rs/<shard_cluster_index>/<shard_number>`
```

Comment:
We do agree that follow-up work will enable the user to pass `shard_cluster_index` and `shard_number` instead of a pubsub topic, right?